### DISCUSSION TITLE
Number of parking spaces quest

### DISCUSSION THREAD

**Discussion by NathanARF** [streetcomplete/StreetComplete#5033] [Ideas]
Hi, had an idea to add a quest to car parks. A quest asking how many total spaces there are in a car park might be a bit much for the average user, as they'd have to count a lot of spaces for larger car parks, but what about a quest asking "How many accessible parking spaces are there here?". This would be a good quest for accessibility. Perhaps a caveat could be implemented for multi-storey car parks asking to check each level.

**Comment by FloEdelmann** [comment]
This was already rejected in https://github.com/streetcomplete/StreetComplete/issues/1106
